File: Iniciante/1117 - Validação de Nota/JavaScript/1117.js
function main() {
  const input = require("fs").readFileSync(0, "utf-8").trim().split("\n");

  let sumOfGrades = 0;
  let counter = 0;
  let i = 0;

  while (counter < 2) {
    let grade = parseFloat(input[i]);
    i++;

    if (grade >= 0 && grade <= 10) {
      sumOfGrades += grade;
      counter++;
    } else {
      console.log("nota invalida");
    }
  }

  console.log("media = " + (sumOfGrades / 2).toFixed(2));
}

main();

File: Iniciante/1117 - Validação de Nota/Python/1117.py
def main():
    sum_of_grades = 0
    counter = 0

    while counter < 2:
        grade = float(input())

        if 0 <= grade <= 10:
            sum_of_grades += grade
            counter += 1
        else:
            print("nota invalida")

    print(f"media = {sum_of_grades / 2:.2f}")


if __name__ == "__main__":
    main()
